crates/db_views/src/vote_view.rs

```diff
@@ -0,0 +1,195 @@
+use crate::structs::VoteView;
+use diesel::{result::Error, ExpressionMethods, QueryDsl};
+use diesel_async::RunQueryDsl;
+use lemmy_db_schema::{
+  newtypes::{CommentId, PostId},
+  schema::{comment_like, person, post_like},
+  utils::{get_conn, limit_and_offset, DbPool},
+};
+
+impl VoteView {
+  pub async fn list_for_post(
+    pool: &mut DbPool<'_>,
+    post_id: PostId,
+    page: Option<i64>,
+    limit: Option<i64>,
+  ) -> Result<Vec<Self>, Error> {
+    let conn = &mut get_conn(pool).await?;
+    let (limit, offset) = limit_and_offset(page, limit)?;
```

Comment:
This uses a default limit of 10 and maximum 50. As the vote objects are quite small it would make sense to use higher values here.

Reply:
Hrm... I'd rather not have a custom default just for this one specifically. The front ends can send a custom one anyway.